### PR TITLE
Issue740 merge ibpsa

### LINKED ISSUE #740: Release v0.9
We should perform a merge of IBPSA Lib into AixLib soon.  
IBPSA now also have its first conversion script located [here](https://github.com/ibpsa/modelica-ibpsa/tree/master/IBPSA/Resources/Scripts/Dymola).

- [x] Take care that IBPSA's CS is fired also in AixLib and of course that the file is copied to AixLib repo.
- [x] Merge IBPSA into AixLib.
- [x] Add license changes manual #717 
- [x] Move packages to `Obsolete` folder acc. to #746 
- [x] Revise [the wiki section](https://github.com/RWTH-EBC/AixLib/wiki/AixLib-and-Modelica-IBPSA-Library-(former-Annex-60)) where necessary and the paragraph below.
- [x] Resolve issues with CI #781 
- [x] Integrate #724 for clearer heatpump and chiller seperation 
- [x] Try to solve errors (see this [comment](https://github.com/RWTH-EBC/AixLib/pull/774#issuecomment-551215184)) in AixLib bevore merging `dev` into `master` and propose changes to `IBPSA`.

### PR DESCRIPTION
**Pull request by DaJansenGit**
With this PR the current version of IBPSA is merged into AixLib

closes #740

### CODE REVIEW

File: AixLib/Resources/Scripts/Dymola/ConvertIBPSA_from_3.0_to_4.0.mos [hunk 1]
```diff
@@ -0,0 +1,67 @@
+// Conversion script for IBPSA library
+//  from: Version 3.0
+//  to:   Version 4.0
+clear
+
+convertClear();
+
+convertClass("IBPSA.Fluid.Sources.FixedBoundary",
```
Comment (PMehrfeld) [inline AixLib/Resources/Scripts/Dymola/ConvertIBPSA_from_3.0_to_4.0.mos]: `IBPSA.` must be replaced by `AixLib.`

Comment (DaJansenGit) [reply AixLib/Resources/Scripts/Dymola/ConvertIBPSA_from_3.0_to_4.0.mos]: the ConvertIBPSA_from_3.0_to_4.0.mos was obsolete but you are right, the v0.11 to v0.8 script wasn't linked in package.mo with 99e8eae this should be solved. Thanks for your review.

**Comment by DaJansenGit** [comment]
We have to wait with the merge for the discussion in #689 because we moved `AixLib.Utilities.HeatTransfer.HeatToStar` model to obsolete and a view models still use it. If this is done we can merge (if the CI runs through 🤞 )

**Comment by DaJansenGit** [comment]
@PMehrfeld all good things come to an end ... 😃 This should finaly be done, could you please add your review. Thanks!

**Comment by PMehrfeld** [comment]
@DaJansenGit: Thanks for this huge bunch of work!
I will do some remaining checks and afterwards I am going to merge this into `dev`.
However, this does not yet close #740, since there are still some ToDos to take care of!

**Comment by PMehrfeld** [comment]
I still receive these 5 errors:  
image

Before merging, we need to solve the errors.  
@DaJansenGit: Either you can have a look at this within the next week or I will see what I can do after 18 Nov.

- [ ] Investigate errors

**Comment by DaJansenGit** [comment]
@PMehrfeld Nothing to be sorry for. 
All these errors come from the IBPSA library itself. Find a screenshot of the Dymola check for the latest master of IBPSA core library.
image

**Pull request merged**